manticore/core/executor.py

```diff
@@ -561,7 +561,7 @@ def fork(self, current_state, symbolic, vals, setstate=None):
         children = []
         if len(vals) == 1:
             constraint = symbolic == vals[0]
-            current_state.add(constraint, check=True) #We already know it's sat
+            current_state.constrain(constraint)
```

Comment:
inline the constraint formula? it's inlined in all other invocations and with new naming, reads a tad redundant.